registry/example/example_test.go

```diff
@@ -0,0 +1,69 @@
+package example
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	eh "github.com/looplab/eventhorizon"
+	eventstore "github.com/looplab/eventhorizon/eventstore/memory"
+	eventpublisher "github.com/looplab/eventhorizon/publisher/local"
+
+	"github.com/looplab/eventhorizon/registry"
+)
+
+// TestDomain bad integration test
+func TestDomain(t *testing.T) {
+	eventStore := eventstore.NewEventStore()
+	eventPublisher := eventpublisher.NewEventPublisher()
+
+	// eventbus := local.
+	r, err := registry.NewBuilder().
+		AddAggregate(NewHereAggregate).
+		AddEvent(func() eh.EventData { return &HereCreated{} }).
+		AddEvent(func() eh.EventData { return &HereApproved{} }).
+		Build(eventStore, eventPublisher)
+
+	if err != nil {
+		t.Error(err)
+		return
+	}
+
+	// test to see if we can create the example aggregate
+	log.Println("Before CreateAggregate")
+	agg, err := eh.CreateAggregate("HereAggregate", eh.NewUUID())
```

Comment:
Loading aggregates manually (here and below) would almost never happen in client code, maybe make a note about that or remove it from the example. It can give the wrong idea for a newcomer.